.travis.yml

```diff
@@ -1,15 +1,12 @@
 language: ruby
 rvm:
-  - 1.9.3
-  - 2.0.0
-  - 2.1.10
-  - 2.3.4
-  - 2.4.3
-  - 2.5.0
-  - jruby-9.1.15.0
+  - 2.3
+  - 2.4
+  - 2.5
+  - jruby-9.2
```

Comment:
I do not see jruby-9.2 on the list of rubies for Travis and the [build](https://travis-ci.org/ruby-git/ruby-git/jobs/396541130) shows that can't find it. 

Travis Rubies: http://rubies.travis-ci.org/

Reply:
Good point, I filed https://github.com/rvm/rvm/pull/4414 about it.

The Travis list usually don't include all the RVM versions, but either way, `jruby-9.2` is not a valid RVM version string in this case.